packages/components/psammead-media-player/src/Amp/index.jsx

```diff
@@ -41,9 +52,15 @@ AmpMediaPlayer.propTypes = {
   placeholderSrc: string.isRequired,
   placeholderSrcset: string,
   title: string.isRequired,
+  alt: string,
```

Comment:
Should this be a required prop?

Reply:
Also, the `width` and `height` are required in the `AmpImg` component.

Reply:
Added the alt, width and height as optional props because the `<AmpMediaPlayer />` seems to work without them - as it's getting the height and width from the `StyledVideoContainer` and `StyledAudioContainer`